### PR TITLE
Suggestions for Language switcher

### PR DESCRIPTION
**Pull request by lalitpatel**
Related to #48

Three suggestions:

1. We can either keep below the credits section;
1. Or, on the top right if we chose it to be short switcher;
1. Else at the bottom if we want a more descriptive one.

image

The drop-downs on the tech tree have fixed positions
sample

### CODE REVIEW

**Comment by HSZemi** [comment]
How do those look on mobile? While we are not targeting mobile devices specifically, it would be nice if they could at least have access to all features.

File: style.css [hunk 1]
```diff
@@ -196,15 +201,12 @@ select::-ms-expand {
 }
 
 .styled_select:after {
-    content: '▸';
-    -webkit-transform: rotate(90deg);
-    -moz-transform: rotate(90deg);
-    -ms-transform: rotate(90deg);
-    transform: rotate(90deg);
+    content: '▾';
```
Comment (HSZemi) [inline style.css]: :sweat_smile:

Comment (lalitpatel) [reply style.css]: 🙈

**Comment by lalitpatel** [comment]
LGTM on mobile

mobile

**Comment by pardoman** [comment]
I'll grab whichever you want @HSZemi , thou I like the top-right one.

**Comment by HSZemi** [comment]
Ok, so I would prefer it to be in the left sidebar. After all, that is a setting that you use *once* at most, so it does not have to be accessible all the time. Sadly, that takes away from the precious vertical space in the sidebar. I guess we would have to sacrifice at least the key for it.

The bottom one has positioning issues, overlapping with the left sidebar on mobile and with the scrollbar on desktop.
grafik

The top right version looks least obtrusive currently, but as stated, there is no need to have it visible all the time.

**Comment by HSZemi** [comment]
We have a working language switcher now, so I am closing this pull request. Thank you for putting in the effort and creating a part of it :yellow_heart:

**Pull request closed without merging**